.travis.yml

```diff
@@ -3,4 +3,9 @@ rvm:
   - 2.1.8
   - 2.2.4
   - 2.3.0
-  - jruby
+  - 2.4.0
+  - jruby-9.1.9.0
+
+matrix:
+  allow_failures:
+  - rvm: jruby-9.1.9.0
```

Comment:
We don't have a Travis here anyway, right? So why we change it?

Reply:
I added travis support to this repo (as in the upstream it does use Travis)